hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieStreamSourceV1.scala

```diff
@@ -54,6 +55,9 @@ class HoodieStreamSourceV1(sqlContext: SQLContext,
                            writeTableVersion: HoodieTableVersion)
   extends Source with Logging with Serializable with SparkAdapterSupport {
 
+  private lazy val useNewParquetFileFormat = parameters.getOrElse(HoodieReaderConfig.FILE_GROUP_READER_ENABLED.key(),
+    HoodieReaderConfig.FILE_GROUP_READER_ENABLED.defaultValue().toString).toBoolean
```

Comment:
Use `SparkConfigUtils#getStringWithAltKeys` to avoid referencing `.key()` and `.defaultValue()`